lib/modules/hosts/derpibooru.js

```diff
@@ -0,0 +1,216 @@
+addLibrary('mediaHosts', 'derpibooru', {
+	name: 'Derpibooru',
+	domains: [
+		'derpiboo.ru',
+		'www.derpiboo.ru',
+		'derpibooru.org',
+		'www.derpibooru.org',
+		'trixiebooru.org',
+		'www.trixiebooru.org'
```

Comment:
A bit late but: `showImages` ignores subdomains when checking the domain, so `www.` can be removed...I'll add a note to `example.js`.